src/__tests__/Carousel.js

```diff
@@ -1,42 +1,42 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import { shallow, mount } from 'enzyme';
```

Comment:
It looks like you also re-formatted a lot of the code with your local editor settings. Could you undo all the whitespace changes and just retain the actual code changes?

Reply:
Tried to stick just to logic changes with no formatting adjustments in the latest commit.